src/robusta/integrations/slack/sender.py

```diff
@@ -568,25 +568,45 @@ def send_holmes_analysis(
 
         except Exception:
             logging.exception(f"error sending message to slack. {title}")
+    
+    def _resolve_slack_thread(
+        self,
+        finding: Finding,
+        sink_params: SlackSinkParams,
+        thread_ts: Optional[str] = None,
+    ) -> tuple[str, Optional[str]]:
+        
+        channel = ChannelTransformer.template(
```

Comment:
A. Any reason to call this if channel override was provided?
B. Not sure i fully understand how this will work if ChannelTransformer.template will work correctly if labels & annotations aren't pass in that flow.